care/facility/api/serializers/camera_preset.py

```diff
@@ -0,0 +1,52 @@
+from rest_framework import serializers
+from rest_framework.exceptions import ValidationError
+
+from care.facility.api.serializers.bed import AssetBedSerializer
+from care.facility.models import CameraPreset
+from care.users.api.serializers.user import UserBaseMinimumSerializer
+
+
+class CameraPresetSerializer(serializers.ModelSerializer):
+    id = serializers.UUIDField(source="external_id", read_only=True)
+    created_by = UserBaseMinimumSerializer(read_only=True)
+    updated_by = UserBaseMinimumSerializer(read_only=True)
+    asset_bed = AssetBedSerializer(read_only=True)
+
+    class Meta:
+        model = CameraPreset
+        exclude = (
+            "external_id",
+            "deleted",
+        )
+        read_only_fields = (
+            "created_date",
+            "modified_date",
+            "is_migrated",
+            "created_by",
+            "updated_by",
+        )
+
+    def get_asset_bed_obj(self):
+        return self.instance.asset_bed if self.instance else self.context["asset_bed"]
```

Comment:
```suggestion
        return self.instance.asset_bed if self.instance else self.context.get("asset_bed", {})
```

Reply:
is returning an empty dict a good idea? how about returning `None`?